### ISSUE TITLE
Aspect ratios wrong

### ISSUE THREAD

**Issue by Eggo-dotcom** [libretro/ludo#427]
Aspect ratios wrong on the genesis emu and on mednafen pce emu,

Nednafen has a command for correct aspect ratio. Should expose or activate

**Comment by kivutar** [comment]
Hi, can you tell me more how they are wrong?

**Comment by Eggo-dotcom** [comment]
8:7 rather than 4:3 in mednafen specific using option correct aspect ratio fixes this

**Comment by kivutar** [comment]
I see, in fact I get very different results depending on the game I try.

Adventure island seems very rectangular, once we crop overscan, while Air Zonk is very square.

Do you know if RetroArch has the same issue?

I'm trying to understand if the bug is in Ludo or in the libretro port of mednafen.

**Comment by Eggo-dotcom** [comment]
Interesting ill test more and report back in a day or so. I normally dont use retroarch but ill try it and comment back. Mednafen accurate is 100% therfore I actually use official mednafen for tg16 pce. 

Your front end caught my attention a while back. I like the concept of bringing my emulators together under a front end like this, looks great on my htpc in my living room. 

Anyway im off for now, i will take some time and compare retroarch behaviour for this vs ludo.

ps if tou enjoy tg16 and pce do yourself a favor and try the accurate core you will be happy.

**Comment by kivutar** [comment]
I see, trying it now, with the accurate core, and all options set to their default value:

<img width="1072" alt="Screen Shot 2021-06-02 at 16 31 47" src="https://user-images.githubusercontent.com/442722/120456669-08943580-c3c0-11eb-9060-45382c70a1c0.png">

I notice a huge difference in overscan amount between these two games.

<img width="1072" alt="Screen Shot 2021-06-02 at 16 33 17" src="https://user-images.githubusercontent.com/442722/120456921-3da08800-c3c0-11eb-90ac-786c03dd9cc6.png">

They all ouput 256 240 according to logs.

**Comment by kivutar** [comment]
OK this is why:

<img width="1072" alt="Screen Shot 2021-06-02 at 18 06 53" src="https://user-images.githubusercontent.com/442722/120470065-51062000-c3cd-11eb-861f-d89932b80ac0.png">

The default value seem to be 6:5.

And this core at least allows switching to 4:3.

**Comment by Eggo-dotcom** [comment]
Ok so did my own retesting with a from scratch install of ludo. The genesis core does have proper aspect I apologize I thought it was off (dumb of me settings on my personal setup)

The mednafen fast aspects were off and are normal behavior of fast. (never realized that, never use fast core to be honest)


Side loaded mednafen_pce_libretro.dll using ludo, set aspect to 4:3 to give original tv expected ratio all looks good.

So basically maybe consider switching to the accurate core? the fast core is handy on low powered systems however the accurate core needs less power than some other cores you already have and gives a perfect experience with more than just ratio correction.

Now just need to convince you to add 2xsai under ludo settings lol
